### PR TITLE
Add twiglet gem

### PR DESCRIPTION
**Pull request by punitcse**
# What
- move ruby code from another repo.
- Change gem version
- workflow changes

@simplybusiness/application-tooling

### CODE REVIEW

File: twiglet.gemspec [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    # TODO: need to change this to rubygems when we publish
+    gem.metadata['allowed_push_host'] = 'https://gemstash.simplybusiness.io/private'
+  else
+    raise(
+      'RubyGems 2.2 or newer is required to protect against public gem pushes.'
+    )
+  end
+
+  gem.name                  = 'twiglet'
+  gem.version               = '0.1.0'
```
Comment (addersuk) [inline twiglet.gemspec]: It it worth moving this into a version.rb file like other gems?

Comment (punitcse) [reply twiglet.gemspec]: good point

File: README.md [hunk 2]
```diff
@@ -1,2 +1,125 @@
-# twiglet-ruby
```
Comment (sleepyfox) [inline README.md:1]: Boo! Need to keep the tag-line.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,125 @@
...
+```bash
+gem install twiglet
+```
+
+## How to use
+
+Create a new logger like so:
+
+```ruby
+require 'logger'
```
Comment (sleepyfox) [inline README.md]: This needs to change as soon as we've published the gem.

File: example_app.rb [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+require_relative 'lib/twiglet/logger'
+
+PORT = 8080
+
+logger = Logger.new(conf: { service: 'petshop' })
+
+# Start our petshop
+logger.info({
+              event: {
```
Comment (sleepyfox) [inline example_app.rb:11]: Indentation hurts my :eyes:

File: Gemfile.lock [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  specs:
+    simplycop (0.5.4)
+      rubocop (~> 0.80.0)
+      rubocop-rails
+      rubocop-rspec
+
+GEM
+  remote: https://rubygems.org/
+  specs:
+    activesupport (6.0.3.1)
```
Comment (sleepyfox) [inline Gemfile.lock:13]: Is there any way we can get rid of all the dev dependencies from this?

Comment (punitcse) [reply Gemfile.lock:13]: I think it might be simplycop 😢

File: twiglet.gemspec (hunk 1 above)
Comment (sleepyfox) [reply twiglet.gemspec]:
```suggestion
  gem.version               = '1.0.0'
```
Because semver.

**Review by sleepyfox — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1,2 +1,125 @@
-# twiglet-ruby
-Like a log, only smaller.
+# Twiglet: Ruby version
+
```
Comment (sleepyfox) [inline README.md:3]:
```suggestion
Like a log, only smaller.

```

**Pull request merged**